config/initializers/new_framework_defaults_6_0.rb

```diff
@@ -42,4 +42,4 @@
 # Enable the same cache key to be reused when the object being cached of type
 # `ActiveRecord::Relation` changes by moving the volatile information (max updated at and count)
 # of the relation's cache key into the cache version to support recycling cache key.
-# Rails.application.config.active_record.collection_cache_versioning = true
+Rails.application.config.active_record.collection_cache_versioning = true
```

Comment:
I don't see how this could cause an issue, just might make some caches invalid temporarily